diff --git a/src/lib/net/CMakeLists.txt b/src/lib/net/CMakeLists.txt
index a7a7d95e..c30dc5fc 100644
--- a/src/lib/net/CMakeLists.txt
+++ b/src/lib/net/CMakeLists.txt
@@ -6,9 +6,6 @@
 # Apple has to use static libraries because "Use of the Apple-provided OpenSSL
 # libraries by apps is strongly discouraged."
 # https://developer.apple.com/library/archive/documentation/Security/Conceptual/cryptoservices/SecureNetworkCommunicationAPIs/SecureNetworkCommunicationAPIs.html
-if(APPLE)
-  set(OPENSSL_USE_STATIC_LIBS TRUE)
-endif()
 
 find_package(OpenSSL ${REQUIRED_OPENSSL_VERSION} REQUIRED COMPONENTS SSL Crypto)
 
