diff --git a/deploy/CMakeLists.txt b/deploy/CMakeLists.txt
index c84e0b5c..880ca07b 100644
--- a/deploy/CMakeLists.txt
+++ b/deploy/CMakeLists.txt
@@ -24,16 +24,6 @@ if(NOT PACKAGE_VERSION_LABEL)
     set (PACKAGE_VERSION_LABEL "${CPACK_PACKAGE_VERSION}")
 endif()
 
-if(WIN32)
-  include(windows/deploy.cmake)
-elseif(UNIX AND NOT APPLE)
-  include(linux/deploy.cmake)
-elseif(APPLE)
-  include(mac/deploy.cmake)
-else()
-  message(STATUS "UNKNOWN System: ${CMAKE_SYSTEM_NAME}")
-endif()
-
 # Always use "deskflow" for start of name
 set(CPACK_PACKAGE_FILE_NAME "${CMAKE_PROJECT_NAME}-${PACKAGE_VERSION_LABEL}-${OS_STRING}")
 message(STATUS "Package Basename: ${CPACK_PACKAGE_FILE_NAME}")
