diff --git a/plugins/00_dokku-standard/install b/plugins/00_dokku-standard/install
deleted file mode 100755
index 7ca97566..00000000
--- a/plugins/00_dokku-standard/install
+++ /dev/null
@@ -1,69 +0,0 @@
-#!/usr/bin/env bash
-set -eo pipefail
-[[ $DOKKU_TRACE ]] && set -x
-
-if [[ ! -f "$DOKKU_ROOT/HOSTNAME" ]]; then
-  hostname -f >"$DOKKU_ROOT/HOSTNAME"
-fi
-
-if [[ ! -f "$DOKKU_ROOT/VHOST" ]]; then
-  [[ $(dig +short "$(<"$DOKKU_ROOT/HOSTNAME")") ]] && cp "$DOKKU_ROOT/HOSTNAME" "$DOKKU_ROOT/VHOST"
-fi
-
-dokku_path="$(which dokku)"
-
-# temporary hack for https://github.com/dokku/dokku/issues/82
-# redeploys all apps after a reboot
-if [[ $(/sbin/init --version 2>/dev/null) =~ upstart ]]; then
-  cat <<EOF >/etc/init/dokku-redeploy.conf
-description "Dokku app redeploy service"
-
-start on filesystem and started docker
-
-script
-  sleep 2 # give docker some time
-  sudo -i -u dokku $dokku_path ps:restore
-end script
-EOF
-fi
-
-if [[ $(systemctl 2>/dev/null) =~ -\.mount ]]; then
-  cat <<EOF >/etc/systemd/system/dokku-redeploy.service
-[Unit]
-Description=Dokku app redeploy service
-Requires=docker.service
-After=docker.service
-
-[Service]
-Type=simple
-User=dokku
-ExecStart=$dokku_path ps:restore
-
-[Install]
-WantedBy=docker.service
-EOF
-  if command -v systemctl &>/dev/null; then
-    systemctl --quiet reenable dokku-redeploy
-  fi
-fi
-
-mkdir -p /etc/update-motd.d
-cat <<EOF >/etc/update-motd.d/99-dokku
-#!/bin/sh
-
-if [ ! -f "/home/dokku/HOSTNAME" ]; then
-  echo "\nTo avoid cli-based dokku configuration, use the web installer"
-  echo "Debian-based installs should have this running and available"
-  echo "on port 80. You can also install it via the following commands:"
-  echo ""
-  echo "  cd /root/dokku"
-  echo "  sudo make dokku-installer\n"
-fi
-EOF
-chmod +x /etc/update-motd.d/99-dokku
-
-chown dokku:dokku "$DOKKU_ROOT/HOSTNAME"
-
-if [[ -f "$DOKKU_ROOT/VHOST" ]]; then
-  chown dokku:dokku "$DOKKU_ROOT/VHOST"
-fi
diff --git a/plugins/00_dokku-standard/subcommands/version b/plugins/00_dokku-standard/subcommands/version
index 046d9709..a2b024e3 100755
--- a/plugins/00_dokku-standard/subcommands/version
+++ b/plugins/00_dokku-standard/subcommands/version
@@ -6,7 +6,7 @@ source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
 dokku_version_cmd() {
   declare desc="prints dokku version"
   local cmd="version"
-  cat "$DOKKU_ROOT/VERSION" || dokku_log_fail "Unable to determine dokku's version"
+  cat "$DOKKU_DRV/VERSION" || dokku_log_fail "Unable to determine dokku's version"
 }
 
 dokku_version_cmd "$@"
diff --git a/plugins/20_events/install b/plugins/20_events/install
deleted file mode 100755
index 7146f27c..00000000
--- a/plugins/20_events/install
+++ /dev/null
@@ -1,76 +0,0 @@
-#!/usr/bin/env bash
-set -eo pipefail
-[[ $DOKKU_TRACE ]] && set -x
-source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
-
-DOKKU_RSYSLOG_FILTER_DIR=/etc/rsyslog.d
-DOKKU_RSYSLOG_FILTER=$DOKKU_RSYSLOG_FILTER_DIR/99-dokku.conf
-DOKKU_LOGROTATE_FILE=/etc/logrotate.d/dokku
-
-flag_rsyslog_needs_restart=n
-
-# This can be done unconditionally as mkdir -p
-# exits gracefully if the path already exists
-# shellcheck disable=SC2174
-mkdir -m 775 -p "$DOKKU_LOGS_DIR"
-case "$DOKKU_DISTRO" in
-  arch | debian | centos | rhel)
-    chgrp dokku "$DOKKU_LOGS_DIR"
-    ;;
-  *)
-    chown syslog:dokku "$DOKKU_LOGS_DIR"
-    ;;
-esac
-
-if [[ ! -f "$DOKKU_EVENTS_LOGFILE" ]]; then
-  touch "$DOKKU_EVENTS_LOGFILE"
-  case "$DOKKU_DISTRO" in
-    arch | debian | centos | rhel)
-      chgrp dokku "$DOKKU_EVENTS_LOGFILE"
-      ;;
-    *)
-      # chown syslog:root might not work on SUSE
-      chown syslog:dokku "$DOKKU_EVENTS_LOGFILE"
-      ;;
-  esac
-  chmod 664 "$DOKKU_EVENTS_LOGFILE"
-fi
-
-if [[ -d "$DOKKU_RSYSLOG_FILTER_DIR" && ! -f "$DOKKU_RSYSLOG_FILTER" ]]; then
-  cat >"$DOKKU_RSYSLOG_FILTER" <<EOF
-:syslogtag, contains, "dokku" $DOKKU_EVENTS_LOGFILE
-EOF
-  flag_rsyslog_needs_restart=y
-fi
-
-if [[ ! -f "$DOKKU_LOGROTATE_FILE" ]]; then
-  cat >"$DOKKU_LOGROTATE_FILE" <<EOF
-$DOKKU_LOGS_DIR/*.log {
-        daily
-        rotate 7
-        missingok
-        notifempty
-        su syslog dokku
-        compress
-        delaycompress
-        postrotate
-                reload rsyslog >/dev/null 2>&1 || true
-        endscript
-        create 664 syslog dokku
-}
-EOF
-
-  if [[ "$DOKKU_DISTRO" == "debian" ]]; then
-    sed -i 's/ syslog dokku$/ root dokku/g' $DOKKU_LOGROTATE_FILE
-  fi
-
-  flag_rsyslog_needs_restart=y
-fi
-
-if [[ -f "$DOKKU_RSYSLOG_FILTER" && "$flag_rsyslog_needs_restart" == "y" ]]; then
-  service rsyslog restart
-fi
-
-[[ ! "$DOKKU_EVENTS" ]] || dokku_log_plugn_trigger_call "$(basename "$0")" "$@"
-
-exit 0
diff --git a/plugins/common/functions b/plugins/common/functions
index 9478c2d7..1437df46 100755
--- a/plugins/common/functions
+++ b/plugins/common/functions
@@ -4,7 +4,7 @@ set -eo pipefail
 
 has_tty() {
   declare desc="return 0 if we have a tty"
-  if [[ "$(/usr/bin/tty || true)" == "not a tty" ]]; then
+  if [[ "$(tty || true)" == "not a tty" ]]; then
     return 1
   else
     return 0
@@ -895,14 +895,14 @@ internal_get_app_urls() {
         local APP_CONTAINER_PORTS="$(get_container_ports "$APP")"
         local app_port
         for app_port in $APP_CONTAINER_PORTS; do
-          echo "$(<"$DOKKU_ROOT/HOSTNAME"):$app_port (container)"
+          echo "$(<"$DOKKU_DRV/HOSTNAME"):$app_port (container)"
         done
       else
         local DOKKU_APP_LISTENERS PORT
         DOKKU_APP_LISTENERS="$(plugn trigger network-get-listeners "$APP" | xargs)"
         for DOKKU_APP_LISTENER in $DOKKU_APP_LISTENERS; do
           PORT="$(echo "$DOKKU_APP_LISTENER" | cut -d ':' -f2)"
-          echo "$SCHEME://$(<"$DOKKU_ROOT/HOSTNAME"):$PORT (container)"
+          echo "$SCHEME://$(<"$DOKKU_DRV/HOSTNAME"):$PORT (container)"
         done
         shopt -u nullglob
       fi
@@ -911,14 +911,14 @@ internal_get_app_urls() {
       for port_map in $DOKKU_PROXY_PORT_MAP; do
         local scheme="$(awk -F ':' '{ print $1 }' <<<"$port_map")"
         local listen_port="$(awk -F ':' '{ print $2 }' <<<"$port_map")"
-        echo "$scheme://$(<"$DOKKU_ROOT/HOSTNAME"):$listen_port"
+        echo "$scheme://$(<"$DOKKU_DRV/HOSTNAME"):$listen_port"
       done
     elif [[ -n "$RAW_TCP_PORTS" ]]; then
       for p in $RAW_TCP_PORTS; do
-        echo "http://$(<"$DOKKU_ROOT/HOSTNAME"):$p"
+        echo "http://$(<"$DOKKU_DRV/HOSTNAME"):$p"
       done
     else
-      echo "$SCHEME://$(<"$DOKKU_ROOT/VHOST")"
+      echo "$SCHEME://$(<"$DOKKU_DRV/VHOST")"
     fi
   fi
 }
diff --git a/plugins/domains/functions b/plugins/domains/functions
index 83d17e8e..dbedfc77 100755
--- a/plugins/domains/functions
+++ b/plugins/domains/functions
@@ -32,7 +32,7 @@ domains_setup() {
   verify_app_name "$1"
   local APP="$1"
   local APP_VHOST_PATH="$DOKKU_ROOT/$APP/VHOST"
-  local GLOBAL_VHOST_PATH="$DOKKU_ROOT/VHOST"
+  local GLOBAL_VHOST_PATH="$DOKKU_DRV/VHOST"
   local RE_IPV4="$(get_ipv4_regex)"
   local RE_IPV6="$(get_ipv6_regex)"
   local DEFAULT_VHOSTS="$(get_default_vhosts "$APP")"
@@ -154,7 +154,7 @@ domains_enable() {
 
 domains_add_global() {
   declare desc="add list of global domains"
-  local GLOBAL_VHOST_PATH="$DOKKU_ROOT/VHOST"
+  local GLOBAL_VHOST_PATH="$DOKKU_DRV/VHOST"
 
   touch "$GLOBAL_VHOST_PATH"
   for DOMAIN in "$@"; do
@@ -178,7 +178,7 @@ domains_add_global() {
 
 domains_remove_global() {
   declare desc="remove list of domains"
-  local GLOBAL_VHOST_PATH="$DOKKU_ROOT/VHOST"
+  local GLOBAL_VHOST_PATH="$DOKKU_DRV/VHOST"
 
   touch "$GLOBAL_VHOST_PATH"
   for DOMAIN in "$@"; do
@@ -189,7 +189,7 @@ domains_remove_global() {
 
 domains_set_global() {
   declare desc="set list of global domains"
-  local GLOBAL_VHOST_PATH="$DOKKU_ROOT/VHOST"
+  local GLOBAL_VHOST_PATH="$DOKKU_DRV/VHOST"
 
   touch "$GLOBAL_VHOST_PATH"
   for DOMAIN in "$@"; do
@@ -220,8 +220,8 @@ get_app_domains() {
   verify_app_name "$1"
   local APP=$1
   local APP_VHOST_FILE="$DOKKU_ROOT/$APP/VHOST"
-  local GLOBAL_VHOST_PATH="$DOKKU_ROOT/VHOST"
-  local GLOBAL_HOSTNAME_PATH="$DOKKU_ROOT/HOSTNAME"
+  local GLOBAL_VHOST_PATH="$DOKKU_DRV/VHOST"
+  local GLOBAL_HOSTNAME_PATH="$DOKKU_DRV/HOSTNAME"
 
   if [[ "$(is_app_vhost_enabled "$APP")" == "true" ]]; then
     if [[ -f "$APP_VHOST_FILE" ]]; then
@@ -263,7 +263,7 @@ get_default_vhosts() {
 
 get_global_vhosts() {
   declare desc="return global vhosts"
-  local GLOBAL_VHOST_FILE="$DOKKU_ROOT/VHOST"
+  local GLOBAL_VHOST_FILE="$DOKKU_DRV/VHOST"
   [[ -f "$GLOBAL_VHOST_FILE" ]] && local GLOBAL_VHOSTS=$(<"$GLOBAL_VHOST_FILE")
   echo "$GLOBAL_VHOSTS"
 }
diff --git a/plugins/nginx-vhosts/functions b/plugins/nginx-vhosts/functions
index f7edb518..42d6badc 100755
--- a/plugins/nginx-vhosts/functions
+++ b/plugins/nginx-vhosts/functions
@@ -325,7 +325,7 @@ nginx_build_config() {
       if [[ "$IS_APP_VHOST_ENABLED" == "true" ]]; then
         local SSL_VHOSTS=$(egrep "^${SSL_HOSTNAME_REGEX}$" "$VHOST_PATH" || true)
       else
-        local SSL_VHOSTS=$(<"$DOKKU_ROOT/HOSTNAME")
+        local SSL_VHOSTS=$(<"$DOKKU_DRV/HOSTNAME")
       fi
       local SSL_SERVER_NAME
       local host
